ghzdac/__init__.py

```diff
@@ -34,6 +34,8 @@
                         cosinefilter, gaussfilter, flatfilter)
 import keys
 import calibrate
+import logging
+logging.basicConfig(level=logging.ERROR)
```

Comment:
Does this affect logging config globally? I'm not too familiar with python logging, but if this does in fact affect global state, then it should probably happen in the `main` function of whatever program is running, instead of during package import.